script/sync

```diff
@@ -134,6 +134,30 @@ def sync_ruby(version)
   puts "done."
 end
 
+def sync_node(version)
+  s3_key = "nodes/Darwin/#{s3_cellar}#{os}/#{version}.tar.bz2"
```

Comment:
Hi, mmm. I'm not really sure... I took the idea from the ruby sync function.
In the rewrite I proposed, the provider is used in the nodebuild provider.
https://github.com/platanus/puppet-nodejs/blob/rewrite/lib/puppet/provider/nodejs/nodebuild.rb#L75

Do you think there is something wrong??